### ISSUE TITLE
Gravações

### ISSUE THREAD

**Issue by rgazetta** [ibinetwork/IssabelBR#7]
Mudou algo, a falha das gravações voltou, quando esta em portugues nao é possível ouvir a gravação.

**Comment by ibinetwork** [comment]
Provavelmente você atualizou o Issabel depois te ter executado o Patch-br.

**Comment by rgazetta** [comment]
Não, basta instalar a versão 20181002 que o patch não corrige a falha das gravações entre outras coisas.

**Comment by ibinetwork** [comment]
Sei que já passou um tempo, por favor relate qual o problema acontece nas gravações e entre outras coisas por favor.

**Comment by rgazetta** [comment]
Na verdade o sistema mesmo atualiza o issabel, e ao fazer isso ocorre falhas, o menu de gravações não funciona, e outros locais ficam desconfigurados, a solução foi remover a atualização do issabel do script no momento.
